src/repositorios/injury.test.js

```diff
@@ -25,7 +26,8 @@ describe(
                                         id: expect.any(Number),
                                         nome: "Short",
                                         detalhes: "Long",
-                                        grade: 100
+                                        grade: 100,
+                                        id_collection: 1
```

Comment:
Test might fail if collection doesn't exist. Fix it using https://github.com/CRICDatabase/searchable-image-database-nodejs/issues/136.